main/usb_uart_bridge_main.c

```diff
@@ -16,13 +16,25 @@
 #include "esp_system.h"
 #include "esp_timer.h"
 #include "rom/ets_sys.h"
+#include "soc/rtc_cntl_reg.h"
 
 #include "tinyusb.h"
 #include "tusb.h"
 #include "tusb_cdc_acm.h"
 
 static const char *TAG = "USB2UART";
 
+// To allow entering the ESP32 bootloader *without* RTS/DTR, set these baudrates on the
+// serial port in this order. Some OSes reset the serial port to a default baudrate so
+// as long as these are opened in this order (but not necessarily sequentially), the
+// bootloader will be triggered.
+const uint32_t MAGIC_BOOTLOADER_TRIGGER_BAUDRATES[] = {
+    111111,
+    222222,
+    333333,
+};
```

Comment:
Is there any chance that common serialport libraries in certain languages don't accept those non-standard baudrates at all?

Reply:
Sure, let's do it. I didn't even know they went below 9600 🤣:

```
B0
B50
B75
B110
B134
B150
B200
B300
B600
B1200
B1800
B2400
B4800
B9600
B19200
B38400
B57600
B115200
B230400
```